Doc/library/socketserver.rst

```diff
@@ -433,7 +433,7 @@ Request Handler Objects
 
    The :attr:`rfile` attributes of both classes support the
    :class:`io.BufferedIOBase` readable interface, and
-   :attr:`DatagramRequestHandler.wfile` supports the
+   the :attr:`wfile` attributes of both classes support the
```

Comment:
There is no need to repeat that part again in the same sentence; it is perfectly clear as it is written.

Reply:
There *is* a need because since Python 3.6 this statement obsolete, now the `StreamRequestHandler.wfile` attribute also supports the `io.BufferedIOBase` writable interface (cf. [`socketserver` documentation](https://docs.python.org/3/library/socketserver.html#socketserver.StreamRequestHandler)):

> Changed in version 3.6: StreamRequestHandler.wfile also supports the [io.BufferedIOBase](https://docs.python.org/3/library/io.html#io.BufferedIOBase) writable interface.

Reply:
Better then to change that sentence like this and include it in the previous paragraph:

```
The :attr:`!rfile` attributes support the :class:`io.BufferedIOBase` readable interface,
and :attr:`!wfile` attributes support the :class:`!io.BufferedIOBase` writable interface.
```

Reply:
+1. Done.